nilearn/decoding/decoder.py

```diff
@@ -1201,7 +1201,7 @@ def __init__(
 
 
 @fill_doc
-class DecoderRegressor(_BaseDecoder):
+class DecoderRegressor(_BaseDecoder, MultiOutputMixin, RegressorMixin):
```

Comment:
From the top of the head, the mixins need to be on the most left to not have issue with the MRO (old discussions: https://github.com/scikit-learn/scikit-learn/pull/8022#issuecomment-267091483)